pyproject.toml

```diff
@@ -0,0 +1,26 @@
+[tool.poetry]
+name = "new-train-tracker"
+version = "1.0.0"
```

Comment:
```suggestion
version = "1.0.0"
```
Should this match the git tag version?

Reply:
It could, but doesn't really matter 👍 The version number here has nearly no function

Reply:
no worries, the `package.json` matches which is probably more important - certainly not going to block merging on this